packages/text-annotator/src/highlight/HighlightPainter.ts

```diff
@@ -17,11 +17,13 @@ export type HighlightPainter = {
     bg: CanvasRenderingContext2D,
     fg: CanvasRenderingContext2D,
     isSelected?: boolean,
-    style?: DrawingStyle | ((annotation: TextAnnotation, selected?: boolean) => DrawingStyle)
```

Comment:
`DrawingStyle` comes from the `A9S`, but for the text annos, there's ` HighlightStyle` type defined below: https://github.com/recogito/text-annotator-js/blob/054ca425b2957c9184565595ff40bfbd5c26a835/packages/text-annotator/src/highlight/HighlightPainter.ts#L31
https://github.com/recogito/text-annotator-js/blob/054ca425b2957c9184565595ff40bfbd5c26a835/packages/text-annotator/src/highlight/HighlightPainter.ts#L44-L56